samples/net/sockets/big_http_download/README.rst

```diff
@@ -36,6 +36,11 @@ Build the Zephyr version of the application like this:
    :goals: build
    :compact:
 
+.. warning::
+
+   On some boards the stack size CONFIG_MAIN_STACK_SIZE must be increased!
```

Comment:
This is not an ok warning to put in the docs. If the stack size needs to be increased, let's increase it to safe enough value. (The fact that Zephyr stack size requirements in Zephyr is constantly grows is a known matter, I raised concern of it, but nobody seems to pay attention.)